### ISSUE TITLE
Upgrade from 2.1.0 to 2.1.1 removed Laminas-Form from modules.config.php

### ISSUE THREAD

**Issue by williamgall** [laminas/laminas-mvc-form#15]
### BC Break Report

After running composer update it removes Laminas-Form from the modules.config.php file

|    Q        |   A
|------------ | ------
| Version     | 2.1.1

#### Summary

After composer updated there was no Laminas-Form in my modules.config.php
#### Previous behavior

Laminas-Form was in my modules.config.php

#### Current behavior

removed Laminas-Form

#### How to reproduce

Run composer update. I have tested this in two separate projects with the same results.

**Comment by Xerkus** [comment]
form provides its own entry.
Can you try to remove vendor directory and run composer install, see if it is added back?

**Comment by Xerkus** [comment]
When component installer plugin processes package updates on composer update it is only invoked for packages that actually changed. Because this package had duplicate entry from laminas-form it ended up removing laminas form module on update.

**Comment by williamgall** [comment]
Yes it adds it back in if you delete the vendor directory and run composer install.  Sorry if this is a false report this is the first time composer update has ever broken my apps.

**Comment by Xerkus** [comment]
This was a bug fixed in this package. It should not have had component installer entries in the first place. This is potentially a bug in laminas-component-installer and it is definitely an edge case but I am not sure it can be fixed.
